model/src/main/kotlin/config/CopyrightGarbage.kt

```diff
@@ -0,0 +1,7 @@
+package com.here.ort.model.config
+
+import java.util.SortedSet
+
+data class CopyrightGarbage(val items: SortedSet<String> = sortedSetOf()) {
+    constructor(vararg items: String) : this(items.toSortedSet())
```

Comment:
Why these two constructors? The `vararg` is implicitly turned into a list (which can be turned into a set).